### PR TITLE
fix build warnings

### PR DESCRIPTION
**Pull request by mairaw**
An attempt to reduce the number of warnings from our build report.

### CODE REVIEW

File: xml/Microsoft.VisualBasic.Compatibility.VB6/BaseControlArray.xml [hunk 1]
```diff
@@ -680,7 +680,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Design environments typically use this method to start the initialization of a component that is used on a form or used by another component. The <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit> method ends the initialization. Using the `BeginInit` and <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit> methods prevents the control from being used before it is fully initialized. The initialization occurs at run time.  
+ Design environments typically use this method to start the initialization of a component that is used on a form or used by another component. The <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit%2A> method ends the initialization. Using the `BeginInit` and <xref:Microsoft.VisualBasic.Compatibility.VB6.BaseControlArray.System#ComponentModel#ISupportInitialize#EndInit%2A> methods prevents the control from being used before it is fully initialized. The initialization occurs at run time.
```
Comment (rpetrusha) [inline xml/Microsoft.VisualBasic.Compatibility.VB6/BaseControlArray.xml]: Both links are still broken, and I'm not sure why. I also tried changing them in a PR a few months ago, and could neither the original syntax that you changed nor the addition of %2A seemed to work.

**Review by BillWagner — APPROVED** [review]
I see fewer warnings in the build reports, so I'll call this a success.

You can :shipit: when you are ready.

**Comment by rpetrusha** [comment]
There are still build warnings here, though. I'll list them in a few minutes, but you might want to hold off on merging, @mairaw.

File: xml/System.Activities/CodeActivity.xml [hunk 2]
```diff
@@ -106,7 +106,7 @@
...
+The workflow runtime uses the default implementation of <xref:System.Activities.CodeActivity.CacheMetadata%2A> to ensure correct construction of a workflow as well as to manage runtime relationships and lifetime rules. The default implementation of <xref:System.Activities.CodeActivity.CacheMetadata%2A> examines the public members of the activity type using the type’s <xref:System.ComponentModel.TypeDescriptor>. These public members are of type <xref:System.Activities.Argument>, <xref:System.Activities.Variable>, [IEnumerable\<Variable>](xref:T:System.Collections.Generic.IEnumerable%601), <xref:System.Activities.Activity>, [IEnumerable\<Activity>](xref:T:System.Collections.Generic.IEnumerable%601), or <xref:System.Activities.ActivityDelegate>. You can override this method to customize the building of the activity’s run-time description and to provide custom validation logic. Any exceptions that are thrown from <xref:System.Activities.CodeActivity.CacheMetadata%2A> are not treated as validation errors. These exceptions will escape from the call to <xref:System.Activities.Validation.ActivityValidationServices.Validate%2A> and must be handled by the caller.
```
Comment (rpetrusha) [inline xml/System.Activities/CodeActivity.xml]: There are two broken links here; they both begin **\<xref:T:**.

File: xml/System.Activities/NativeActivity.xml [hunk 3]
```diff
@@ -202,7 +202,7 @@ public sealed class MySequence : NativeActivity
...
+ The workflow runtime uses the default implementation of <xref:System.Activities.NativeActivity.CacheMetadata%2A> to ensure correct construction of a workflow as well as to manage run-time relationships and lifetime rules. The default implementation of <xref:System.Activities.NativeActivity.CacheMetadata%2A> examines the public members of the activity type using the type’s <xref:System.ComponentModel.TypeDescriptor>. These public members are of type <xref:System.Activities.Argument>, <xref:System.Activities.Variable>, [IEnumerable\<Variable>](xref:T:System.Collections.Generic.IEnumerable%601), <xref:System.Activities.Activity>, [IEnumerable\<Activity>](xref:T:System.Collections.Generic.IEnumerable%601), or <xref:System.Activities.ActivityDelegate>. You can override this method to customize the building of the activity’s run-time description and to provide custom validation logic. Any exceptions that are thrown from <xref:System.Activities.NativeActivity.CacheMetadata%2A> are not treated as validation errors. These exceptions will escape from the call to <xref:System.Activities.Validation.ActivityValidationServices.Validate%2A> and must be handled by the caller.
```
Comment (rpetrusha) [inline xml/System.Activities/NativeActivity.xml]: Here also, two broken links beginning **\<xref:T:**.

File: xml/System.Activities/NativeActivity`1.xml [hunk 4]
```diff
@@ -135,7 +135,7 @@
...
+ The workflow runtime uses the default implementation of <xref:System.Activities.NativeActivity%601.CacheMetadata%2A> to ensure correct construction of a workflow as well as to manage run-time relationships and lifetime rules. The default implementation of <xref:System.Activities.NativeActivity%601.CacheMetadata%2A> examines the public members of the activity type using the type’s <xref:System.ComponentModel.TypeDescriptor>. These public members are of type <xref:System.Activities.Argument>, <xref:System.Activities.Variable>, [IEnumerable\<Variable>](xref:T:System.Collections.Generic.IEnumerable%601), <xref:System.Activities.Activity>, [IEnumerable\<Activity>](xref:T:System.Collections.Generic.IEnumerable%601), or <xref:System.Activities.ActivityDelegate>. You can override this method to customize the building of the activity’s run-time description and to provide custom validation logic. Any exceptions that are thrown from <xref:System.Activities.NativeActivity%601.CacheMetadata%2A> are not treated as validation errors. These exceptions will escape from the call to <xref:System.Activities.Validation.ActivityValidationServices.Validate%2A> and must be handled by the caller.
```
Comment (rpetrusha) [inline xml/System.Activities/NativeActivity`1.xml]: This also has broken **\<xref:T:** links.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Some additional broken links, @mairaw. There are a number that begin **\<xref:T:** that are easy fixes. I'm not sure how to fix the Visual Basic broken links; we may just have to live with their being broken.

File: xml/Microsoft.VisualBasic.Compatibility.VB6/BaseControlArray.xml (hunk 1 above)
Comment (mairaw) [reply xml/Microsoft.VisualBasic.Compatibility.VB6/BaseControlArray.xml]: 😢 

ok, I'll create a bug for this then

**Review by mairaw — COMMENTED** [review]
Thanks for your review guys! @rpetrusha I've addressed your comments.

**Comment by rpetrusha** [comment]
This looks really good, and we're down to 55 issues in the build report! Thanks for doing this, @mairaw. You can merge when you're ready.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**